vtrace/envitools.py

```diff
@@ -35,27 +42,28 @@ def emuFromTrace(trace):
     rsnap = trace.getRegisterContext().getRegisterSnap()
     emu.setRegisterSnap(rsnap)
 
-    if trace.getMeta('Platform') == 'windows':
+    if plat == 'windows':
         emu.setSegmentInfo(e_i386.SEG_FS, trace.getThreads()[trace.getMeta('ThreadId')], 0xffffffff)
 
     return emu
 
 def lockStepEmulator(emu, trace):
     while True:
-        print "Lockstep: 0x%.8x" % emu.getProgramCounter()
+        print("Lockstep: 0x%.8x" % emu.getProgramCounter())
         try:
             pc = emu.getProgramCounter()
             op = emu.parseOpcode(pc)
             trace.stepi()
             emu.stepi()
             cmpRegs(emu, trace)
-        except RegisterException, msg:
-            print "Lockstep Error: %s: %s" % (repr(op),msg)
-            setRegs(emu, trace)
+        except RegisterException as msg:
+            print("Lockstep Error: %s: %s" % (repr(op), msg))
+            setRegs(emu, trace)  # TODO: Where is this from?
```

Comment:
is this part of your PR?  should it be?  or is this a note which should be tracked down?

Reply:
i don't see it anywhere in the codebase.

Reply:
Oh, my question was a note to ask you or visi "where is this actually defined"? Forgot to ask.

But yea, I couldn't find where this was actually defined either. I'm guessing remove it for now?

Reply:
digging into the "Way back" machine, this used to be defined in vtrace/envitools.py:

```
reg_map = [
    (e_i386.REG_EAX, "eax"),
    (e_i386.REG_ECX, "ecx"),
    (e_i386.REG_EDX, "edx"),
    (e_i386.REG_EBX, "ebx"),
    (e_i386.REG_ESP, "esp"),
    (e_i386.REG_EBP, "ebp"),
    (e_i386.REG_ESI, "esi"),
    (e_i386.REG_EDI, "edi"),
    (e_i386.REG_EIP, "eip"),
    (e_i386.REG_EFLAGS, "eflags")
    ]

#FIXME intel specific
def setRegs(emu, trace):
    for idx,name in reg_map:
        tr = trace.getRegisterByName(name)
        emu.setRegister(idx, tr)


```

Reply:
doing a little research, this was killed by @l1berty back on 11/22/2014

Reply:
Interesting. I'll leave it commented out for now comment it out, and later once we have time to make a more generic setRegs function we can add that back in.